### DISCUSSION TITLE
Is it possible to remove the "language logo" from the language dropdown?

### DISCUSSION THREAD

**Discussion by krillboi** [facebook/docusaurus#4884] [Q&A] [answered]
We have received some feedback that people find the language logo confusing so I have an assignment to remove this from the dropdown. Is this just accomplished with swizzling?

![language_dropdown](https://user-images.githubusercontent.com/59022282/120440913-53905780-c384-11eb-8b86-45eb122f6328.png)

**Comment by slorber** [comment, marked as answer]
Yes you need to swizzle `@theme/IconLanguage`, not much code to maintain on your side

I just ported the existing v1 svg to v2, and not a designer. If you have a better suggestion please let us know :)

**Reply by krillboi** [reply]
Great, thanks!
